.circleci/config.yml

```diff
@@ -32,13 +32,6 @@ commands:
           destination: test-results
 
 jobs:
-  build_2_x:
-    docker:
-      - image: ruby:2.7.8
-    working_directory: ~/repo
-    steps:
-      - build
-
```

Comment:
@thomasLeclaire do you know why this "build_2_x" step still appears in GitHub ? It's gone on CircleCI but still blocks validation on GitHub
Same on this simpler PR https://github.com/Effilab/bing_ads_ruby_sdk/pull/92

Reply:
it is the branches protections : defined there https://github.com/Effilab/bing_ads_ruby_sdk/settings/branch_protection_rules/4793994
just removed the 2.x one.